json_rpc/servers/socketserver.nim

```diff
@@ -18,6 +18,7 @@ export errors, server
 type
   RpcSocketServer* = ref object of RpcServer
     servers: seq[StreamServer]
+    processClientHook*: StreamCallback2
```

Comment:
There is no reason to expose this field to the public.

Reply:
Will add it to the constructor and remove the `*`. The reasoning was to follow what `StreamServer` does.